runhouse/rns/defaults.py

```diff
@@ -279,6 +295,16 @@ def delete_defaults(self, config_path: Optional[str] = None):
         except OSError:
             raise Exception(f"Failed to delete config file from path {config_path}")
 
+    def set_default_launcher_type(self, launcher_type):
```

Comment:
are there individual use cases for the `launcher_type` property setter, and actually setting it in the `default_cache` or could we just combine them and have the setter also update the `default_cache`?

Reply:
im not reallty sure we need it at all actually, this doesn't feel like the kind of thing a user would change all that much. i think i'll removce it

Reply:
the launcher_type setting only sets the variable and not value in the defaults cache though so then there's no way to set it right? do you think this make more sense to propagate through and save it down?
```
@launcher_type.setter
    def launcher_type(self, value):
        self._launcher_type = value
        self.set("launcher_type", value)
```

Reply:
oh i see, yea this is prob a config type we want to persist, so may as well save it down